src/features/swap/hooks/useUnstaking.ts

```diff
@@ -39,7 +39,7 @@ export function useUnstaking() {
       status: 'signed_transaction',
       value: stCeloAmount.displayAsBase(),
     });
-    await api.withdraw(address);
+    api.withdraw(address);
```

Comment:
Are we sure we want to remove await also on withdraw ? 

Once this is done user will not know whether withdrawal succeeded. 

(For activation it doesn't matter that much since we are running it once a day automatically and it is not that vital)

Reply:
This should be fine to do here, as this hook never does anything with the HTTP response. Any failed withdrawal would be handled by `useWithdrawalBot` in `AccountContext.tsx` whenever a page is loaded.

Reply:
Just double checked the error handling (for both `withdraw` and `activate`) by forcing a 500 response from the backend. No errors were shown in the front end when that happens.

Front-end works as expected

Reply:
In that case we could consider to add displaying of errors but if we will decide so it should be in separate task